doc/example-spouse.md

```diff
@@ -88,12 +88,17 @@ Then we compile our application, as we must do whenever we change `app.ddlog`:
 deepdive compile
 ```
 
-Finally, we tell DeepDive to execute the steps to load the `articles` table:
+Finally, we tell DeepDive to execute the steps to load the `articles` table using the `input/articles.tsv.sh` script. You must have the [full corpus](http://research.signalmedia.co/newsir16/signal-dataset.html) downloaded.
 
 ```bash
 deepdive do articles
 ```
 
+Alternatively, a sample of 1000 articles can be loaded directly by issuing the following command:
+```bash
+deepdive load articles input/articles-1000.tsv.bz2
```

Comment:
Agreed. I think I had a script doing this and forgot about it. Built-in shorthand is a great idea